tests/foreman/api/test_new_repository.py

```diff
@@ -0,0 +1,55 @@
+"""Unit tests for the new ``repositories`` paths.
+
+:Requirement: Repository
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: Repositories
+
+:Assignee: chiggins
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from nailgun import entities
+from requests.exceptions import HTTPError
+
+from robottelo.api.utils import enable_rhrepo_and_fetchid
+from robottelo.api.utils import upload_manifest
+from robottelo.constants import PRDS
+from robottelo.constants import REPOS
+from robottelo.constants import REPOSET
+
+
+@pytest.mark.skip_if_open('BZ:2137950')
+@pytest.mark.tier1
+def test_negative_disable_repository_with_cv(module_org, session_entitlement_manifest):
+    """"""
+    upload_manifest(module_org.id, session_entitlement_manifest.content)
```

Comment:
We have fixture available for the same.
https://github.com/SatelliteQE/robottelo/blob/c5b672b80a35d6c0926365c73c75092d3e8a1eac/pytest_fixtures/component/taxonomy.py#L79